students/11th/SangwookRyu/main.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>westagram</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+     <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <!--navigation-->
+    <div class="nav-box">
+      <nav class="nav">
+        <div class="logo-box">
+          <div class="logo1"><img alt = "icon-instagram" src ="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/logo.png"></div>
+          <div class="line"></div>
+          <div class="logo2"><img alt = "logo-instagram" src ="/img/logo_text.png"></div>
```

Comment:
부모 자식 관계가 잘 드러날 수 있도록 줄바꿈 해주세요
```suggestion
          <div class="logo2">
            <img alt = "logo-instagram" src ="/img/logo_text.png">
          </div>
```

Reply:
수정했습니다